### PR TITLE
Issue #141: Emit defaultValue if underlying onSharedPreferenceChanged…

### PR DESCRIPTION
**Pull request by banasiak**
…Listener returns null

### CODE REVIEW

**Comment by audioteka-mobile** [comment]
@f2prateek It looks exactly like what I've done meanwhile - would it be possible for you to make a release with these changes today...?

**Comment by mseroczynski** [comment]
@f2prateek It looks exactly like what I've done meanwhile - would it be possible for you to make a release with these changes today...?

File: rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RxSharedPreferences.java [hunk 1]
```diff
@@ -43,7 +45,11 @@ private RxSharedPreferences(final SharedPreferences preferences) {
         final OnSharedPreferenceChangeListener listener = new OnSharedPreferenceChangeListener() {
           @Override
           public void onSharedPreferenceChanged(SharedPreferences preferences, String key) {
-            emitter.onNext(key);
+            if(key == null) {
```
Comment (f2prateek) [inline rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RxSharedPreferences.java:48]: nit: space after if

**Review by f2prateek — APPROVED** [review]
thanks, this actually looks good to me!

File: rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RxSharedPreferences.java (hunk 1 above)
Comment (banasiak) [reply rx-preferences/src/main/java/com/f2prateek/rx/preferences2/RxSharedPreferences.java]: fixed!

**Review by f2prateek — APPROVED** [review]

**Pull request merged**